run-tests.py

```diff
@@ -48,7 +48,6 @@ def get_args():
 
 def run_sbt_tests(root_dir, test_group, coverage, scala_version=None):
     print("##### Running SBT tests #####")
-    is_running_spark_tests = test_group is None or test_group == "spark"
```

Comment:
@vkorukanti Seems like `is_running_spark_tests` is actually not used anywhere? So I'm removing it.